closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -138,9 +138,12 @@ public final class RestrictedSecurity {
             }
         }
 
-        // If user has specified a profile, use that
-        selectedProfile = System.getProperty("semeru.customprofile");
-        userSetProfile = selectedProfile != null;
+        // If user has specified a profile, use that.
+        String semeruCustomprofile = System.getProperty("semeru.customprofile");
+        if (semeruCustomprofile != null) {
+            selectedProfile = semeruCustomprofile;
+        }
+        userSetProfile = semeruCustomprofile != null;
```

Comment:
"p" should be capitalized; suggest `customProfile` for consistency with other fields/locals.
Also suggest the null check need not be repeated:
```java
        String customProfile = System.getProperty("semeru.customprofile");
        if (customProfile != null) {
            selectedProfile = customProfile;
            userSetProfile = true;
        } else {
            userSetProfile = false;
        }
```

Reply:
Modified